pkgs/development/python-modules/qiskit-aer/default.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, pythonOlder
+, buildPythonPackage
+, fetchFromGitHub
+, cmake
+, cython
+, numpy
+, openblas
+, pybind11
+, scikit-build
+, cvxpy
+  # Check Inputs
+, qiskit-terra
+, stestr
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "qiskit-aer";
+  version = "0.3.4";
+
+  disabled = pythonOlder "3.5";
+
+  src = fetchFromGitHub {
+    owner = "Qiskit";
+    repo = pname;
+    rev = version;
+    sha256 = "0d9s8j06s2dsxq127m21b309pvqb8fj9zia70jc0clsarrawg314";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    cython
+    scikit-build
+  ];
+
+  buildInputs = [
+    openblas
+  ];
+
+  propagatedBuildInputs = [
+    numpy
+    pybind11
+    cvxpy
+  ];
+
+  prePatch = ''
+    # remove dependency on PyPi cmake package, which isn't in Nixpkgs
+    substituteInPlace setup.py --replace ", 'cmake'" ""
+
+    # Add tests & test config to distribution
+    echo "recursive-include test *" >> MANIFEST.in
+    echo "include *.conf" >> MANIFEST.in
```

Comment:
This shouldn't be necessary, we don't create an sdist

Reply:
``scikit-build`` seems to build an sdist then install it. This seemed the easiest way to get the tests into $out.